MessengerOnFileForLinux/DisplayChat/src/InformationPrinter.cpp

```diff
@@ -10,3 +10,39 @@ void InformationPrinter::printInformation(const std::string& informationToPrint)
     refresh();
     std::this_thread::sleep_for(std::chrono::seconds(2));
 }
+
```

Comment:
te wszystkie 3 funkcje są idealnym przykładem duplikacji kodu, zachowaj gdzieś z nich diff to pokażemy na bakcylu

Refaktor:
- namespace z constexpr z wiadomościami
- jedna funkcja, która robi wszystko do "move(2,0)" włącznie, przekazujesz std::initializer_list 
- dedykowana funkcja dla każdego printa, gdzie wywołujesz tą wyżej i końcówkę
- ewentualnie można pokombinować z przekazywaniem też resztej części i miec jedną funkcję, ale nie widzę jak to zrobić czytelnie bez miliona parametrów i ifów